frontend/src/components/EntityBulkAdd.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div v-for="num in entityFormComponentsCountRange" :key="`ef-${num}`">
+    <EntityForm :ref="`entity-form-${num}`" :schema="schema"
```

Comment:
I've stumbled upon sth weird while testing this out. If there's something wrong in certain entity forms, "save all" will only create some of them. After fixing what was wrong I now click "save all" again but since the correct items were already created now there are errors again. I'd expect that the created ones would probably disappear from the list leaving only the errored ones?

If there's no error maybe user can be redirected to the entity list (tho there's a risk that the created entities won't be visible on the page)? Maybe just resetting the bulk add form(s) if there's nothing wrong would be enough?
In either case it feels a bit strange nothing changes after a successful save and user stays in the same tab with all the data that was already sent away and successfully saved.